Jenkinsfile

```diff
@@ -31,7 +31,7 @@ def buildImages = { ->
 }
 
 def getDockerVersions = { ->
-  def dockerVersions = ["19.03.5"]
+  def dockerVersions = ["19.03.11"]
   wrappedNode(label: "ubuntu && !zfs && amd64") {
```

Comment:
It picked Ubuntu 19.10 randomly.
Should we pin it better by using a label `ubuntu-1804`?

Reply:
Or should we remove the ubuntu-1910 machine again? Maybe better and add ubuntu-2004 LTS machines instead. 🤔

Reply:
Yes, probably, or do you have ideas for a labelling scheme where those nodes are not selected in the "regular" pools?

Reply:
oh, our comments crossed LOL

I just opened https://github.com/docker/docker-py/pull/2585